debian/changelog

```diff
@@ -1,3 +1,9 @@
+ceph (17.2.3-1) stable; urgency=medium
+
+  * New upstream release
+
+ -- Ceph Release Team <ceph-maintainers@ceph.com>  Thu, 28 Jul 2022 21:52:09 +0000
```

Comment:
Aren't we using ceph.io now?

Reply:
Good point.  E-mails get forwarded but I fixed the TLD here: https://github.com/ceph/ceph-build/pull/2041

This can't be fixed in this PR or we'd have to start all over with the release.